pkg/idl/doc.go

```diff
@@ -79,13 +79,15 @@ type ListType string
 type ListMapKey string
 
 // MapType annotates a map to further describe its topology. It may
-// have only one value: "atomic". Atomic means that the entire map is
-// considered as a whole, rather than as distinct values.
+// have one of two values: `atomic` or `granular`. `atomic` means that the entire map is
+// considered as a whole; actors that wish to update the map can only
+// entirely replace it. `granular` means that specific values in the map can be
+// updated separately from other fields.
 //
 // By default, a map will be considered as a set of distinct values that
-// can be updated individually. This default WILL NOT generate any
-// openapi extension, as this will also be interpreted as the default
-// behavior in the openapi definition.
+// can be updated individually (i.e. the equivalent of `granular`).
+// This default WILL NOT generate any openapi extension, as this will also
```

Comment:
> Which doc are you referring to?

This doc :-)

Reply:
We talked about it during the meeting, but making my comment "official". I think it's strictly better to have the "granular" explicitly in the openapi than rely on the missing value.

Reply:
Done 👌 (for `structType` too).